src/argus/notificationprofile/notification_media.py

```diff
@@ -38,11 +43,17 @@ def send(incident, user):
             "incident_dict": incident_dict,
             "longest_field_name_length": len(max(incident_dict, key=len)),
         }
-        user.email_user(
-            subject=f"{settings.NOTIFICATION_SUBJECT_PREFIX}{title}",
-            message=render_to_string("notificationprofile/email.txt", template_context),
-            html_message=render_to_string("notificationprofile/email.html", template_context),
-        )
+        try:
+            user.email_user(
+                subject=f"{settings.NOTIFICATION_SUBJECT_PREFIX}{title}",
+                message=render_to_string("notificationprofile/email.txt", template_context),
+                html_message=render_to_string("notificationprofile/email.html", template_context),
+            )
+        except ConnectionRefusedError as e:
+            EMAIL_HOST = getattr(settings, "EMAIL_HOST", "Error: Not set")
+            EMAIL_PORT = getattr(settings, "EMAIL_PORT", "Error: Not set")
+            LOG.error("Notification: Email: Connection refused to \"%s\", port \"%s\"", EMAIL_HOST, EMAIL_PORT)
```

Comment:
I'd prefer if this was changed to
```suggestion
            LOG.error(f'Notification: Email: Connection refused to "{EMAIL_HOST}", port "{EMAIL_PORT}"')
```
to be consistent with the string formatting elsewhere in the code. (Also see the similar comment below.) I'll approve the PR anyway, though 🙂

Reply:
Ah, but logging is magic. Note the lack of the ` % ` building the string.

The string is only created if log level is at or below INFO in this case. With f-strings, it is always made. I keep forgetting, and keep wondering why an innerloop spends so much time on creating debug-statements...